roles/kvm/tasks/grub.yml

```diff
@@ -0,0 +1,32 @@
+---
+##
+# Configures grub menu entry and handles GPU.
+##
+
+- name: Create GRUB iommu.cfg configuration file
+  become: true
+  ansible.builtin.copy:
+    dest: /etc/default/grub.d/iommu.cfg
+    content: |
```

Comment:
Nitpick: maybe it would be better to create a dedicated template file for this cfg, the same way it's done for the `gpu_passthrough_entry`?

Reply:
Done